src/pre_commit_terraform/terraform_docs_replace.py

```
@@ -1,76 +1,97 @@
 def invoke_cli_app(parsed_cli_args: Namespace) -> ReturnCodeType:
+    """
+    Execute main pre-commit hook logic.
+
+    Args:
+        parsed_cli_args: Parsed arguments from CLI.
+
+    Returns:
+        int: The exit code of the hook.
+    """
+
     warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)